Radzen.Blazor/RadzenDialog.razor

```diff
@@ -271,8 +264,14 @@
         string heightStyle = string.IsNullOrEmpty(sideDialogOptions?.Height)
             ? string.Empty
             : $"height: {sideDialogOptions.Height};";
-
-        return $"{widthStyle}{heightStyle}{sideDialogOptions?.Style}";
+        string minWidthStyle = sideDialogOptions?.ResizableMinWidth is double minWidth
```

Comment:
I don't think setting the min-width and min-height style is needed. Try without it and if everything works just use the values passed to the JS function.